spec/controllers/user_spec.rb

```diff
@@ -2,20 +2,28 @@
 
 require 'rails_helper'
 
-RSpec.describe UserController, type: :controller do
+RSpec.describe UsersController, type: :controller do
   describe 'GET #user' do
     context 'when viewing create user page in the browser' do
       it 'returns a 200 OK response' do
-        get :User
+        get :user
         expect(response.status).to eq(200)
       end
     end
   end
-end
+  describe 'POST #create' do
+    context 'When user submits correct params' do
```

Comment:
What happens to the user when they submit incorrect params? It's important to always have test cases for the positive and the negative situations - the happy path and the unhappy path.

Reply:
I can see you've addressed this in the code, feel free to mark as resolved.